.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.9, '3.10', '3.11', '3.12', '3.13.0-rc.2']  # latest release minus two
+        python-version: [3.9, '3.10', '3.11', '3.12', '3.13']  # Starting from earliest supported by Django LTS
```

Comment:
I'd just drop the comment. I think we're essentially supporting the same python versions as Django does for the versions of that which we support. But it's easier not to have a comment.